source/Projectile.cpp

```diff
@@ -378,13 +378,18 @@ void Projectile::BreakTarget()
 // guided missiles.
 void Projectile::CheckLock(const Ship &target)
 {
-	double base = hasLock ? 1. : .15;
+	double relockRate = .3;
```

Comment:
This ought to be a `static const double` (or `static constexpr double`) with its name in SNAKE_CASE, like `RELOCK_RATE` since it's a compile time constant.
```c++
	static const double RELOCK_RATE = .3;
```

A bunch of the other variables in this method could also be marked `const`, but a lot of them go out of scope two lines after they're declared, so it's not really that important.